raiden/utils/serialization.py

```diff
@@ -57,29 +62,29 @@ def deserialize_networkx_graph(data: str) -> networkx.Graph:
 
 
 def serialize_participants_tuple(
-        participants: typing.Tuple[typing.Address, typing.Address],
-) -> typing.List[str]:
+        participants: Tuple[Address, Address],
+) -> List[str]:
     return [
         to_checksum_address(participants[0]),
         to_checksum_address(participants[1]),
     ]
 
 
 def deserialize_participants_tuple(
-        data: typing.List[str],
-) -> typing.Tuple[typing.Address, typing.Address]:
+        data: List[str],
+) -> Tuple[Address, Address]:
     assert len(data) == 2
     return (
         to_canonical_address(data[0]),
         to_canonical_address(data[1]),
     )
 
 
-def serialize_merkletree_layers(data) -> typing.List[str]:
+def serialize_merkletree_layers(data) -> List[str]:
     return map_list(serialize_bytes, data[LEAVES])
 
 
-def deserialize_merkletree_layers(data: typing.List[str]):
+def deserialize_merkletree_layers(data: List[str]):
```

Comment:
I did no fix all function types, just the ones for serialization de-serialization because that is what breaks on my pr.

Reply:
What exactly broke in the PR that made you need generics here? I am curious.

Reply:
This is not using generics. Where I did use generics it's because the function is generic and without the proper type signature type checking fails.